cache-core/src/test/groovy/io/micronaut/cache/jcache/CacheConditionalSpec.groovy

```diff
@@ -0,0 +1,238 @@
+package io.micronaut.cache.jcache
+
+import io.micronaut.cache.annotation.CacheConfig
+import io.micronaut.cache.annotation.CacheInvalidate
+import io.micronaut.cache.annotation.CachePut
+import io.micronaut.cache.annotation.Cacheable
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.annotation.Factory
+import io.micronaut.context.annotation.Requires
+import io.micronaut.core.async.annotation.SingleResult
+import jakarta.inject.Singleton
+import org.reactivestreams.Publisher
+import reactor.core.publisher.Flux
+import spock.lang.AutoCleanup
+import spock.lang.Shared
+import spock.lang.Specification
+
+import javax.cache.CacheManager
+import javax.cache.Caching
+import javax.cache.configuration.MutableConfiguration
+import java.time.Instant
+import java.util.concurrent.CompletableFuture
+
+class CacheConditionalSpec extends Specification {
```

Comment:
there is a lot of duplication in this test.

Reply:
I have removed it with https://github.com/micronaut-projects/micronaut-cache/pull/692/commits/bde8a6efd74842579c5c9f7fbc25ef10c041d1c6